internal/cypress/docker/runner.go

```diff
@@ -234,15 +251,20 @@ func (r *Runner) teardown(logDir string) error {
 }
 
 func (r *Runner) runSuite(suite cypress.Suite) error {
+	var err error
+
 	defer func() {
-		log.Info().Msg("Tearing down environment")
+		if err == nil {
```

Comment:
I wouldn't bother with this (or with `var err error`).

Reply:
If I leave it without checking the error, the order of the error messages will be ERR, INFO, ERR. No escape this and leave only ERR, ERR I would leave this check.

12:24PM ERR Failed to setup test environment error="Docker image name property was not specified in your config file"
12:24PM INFO Tearing down environment
12:24PM ERR 1 suite(s) failed

Reply:
Which is honestly fine, since it was trying to tear down the environment. It's not like you skipped that step entirely ;)

Reply:
I got your point.